src/herder/test/HerderTests.cpp

```diff
@@ -5862,7 +5862,7 @@ TEST_CASE("SCP message capture from previous ledger", "[herder]")
     // Initialize simulation
     auto networkID = sha256(getTestConfig().NETWORK_PASSPHRASE);
     auto simulation = std::make_shared<Simulation>(
-        Simulation::OVER_LOOPBACK, networkID, [version](int i) {
+        Simulation::OVER_LOOPBACK, networkID, [](int i) {
```

Comment:
Unless Microsoft fixed their compiler this unnecessary `version` capture is necessary due to an MSVC bug (see #4194, and specifically [this comment](https://github.com/stellar/stellar-core/pull/4194#issuecomment-1947821687))

Reply:
Ok, fixed (crudely: made it non-constexpr and captured it)